libparsers/src/xml/dynamic_background.cpp

```diff
@@ -41,6 +41,17 @@ namespace xml {
 
 					auto const absolute = get_relative_file(filename, object_filename);
 
+					current_section = current_section->next_sibling();
+					// If multiple object_backgrounds specified add an error.
+					if (current_section != nullptr) {
+						auto const* const is_second_object =
```

Comment:
Delete this and the if statement, you don't need to check to see if there is another object node, if there was a sibling at all, you're done.